controlled_access_terms.info.yml

```diff
@@ -1,7 +1,7 @@
 name: 'Controlled Access Terms'
 type: module
 description: 'Provides topic, geographic, person, family, and corporate body entites as well as an authorities link field.'
-core_version_requirement: ^10.2
+core_version_requirement: ^10 || ^11
```

Comment:
Shouldn't this be `^10.2 || 11`? I think there was a feature we use that requires 10.2 as the minimum.

Reply:
@seth-shaw-asu Thanks. I've restored 10.2 as minimum.